src/list.rs

```diff
@@ -0,0 +1,34 @@
+use handlebars::{handlebars_helper, Handlebars};
+
+pub fn addhelpers(x: &mut Handlebars) {
+    handlebars_helper!(until_step_count: |start: isize, count: isize, step: usize| {
```

Comment:
probably rename `count` to `stop` or `max` here. count seems misleading. what do you think? 

also name of the function could be `until_step` instead of `until_step_count`.

please let me know if i misunderstood something here

Reply:
It makes sense, I will make the changes.